src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -2252,6 +2258,8 @@ private Task<string> ResetConsole()
 
         public bool IsChildProcessDebugging => _childProcessHandler != null;
 
+        public string[] Features { get; private set; }
```

Comment:
nit: instead of exposing an array, use a `IReadOnlyCollection<string>`. You probably also want to initialize it to an empty collection since we will only have it for GDB.

Reply:
lldb doesn't implement that part of the MI?

Reply:
You put your code that populates under an 'if (gdb)' path. I don't know for sure if LLDB supports `-list-features` or not, but your code will not assign to it.

Reply:
Yeah it's probably better as a virtual method in the factory like in the other PR.